### PR TITLE
[library] Web component parameter refactoring

### PR DESCRIPTION
**Pull request by dwnoble**
* Renamed `comparisonVariables` to `variables`. 
* Renamed `comparisonPlaces` to `places`. 
  * In `datacommons-map`, marked `place` as `@deprecated` rather than removing it to avoid breaking existing partners.
* Renamed `place` to `parentPlace` where appropriate.

### CODE REVIEW

**Comment by dwnoble** [comment]
Note: Let's remove `experimental/datacommons-js` in a future PR

File: web-components/docs/README.md [hunk 1]
```diff
@@ -443,21 +454,21 @@ Show a ranking of US States by population, highest to lowest
...
-    title="US States with the Lowest Population"
-    place="country/USA"
-    childPlaceType="State"
-    variable="Count_Person"
-    showLowest=true
+  title="US States with the Lowest Population"
+  parentPlace="country/USA"
+  childPlaceType="State"
+  variable="Count_Person"
+  showLowest="true"
```
Comment (juliawu) [inline web-components/docs/README.md:472]: Note for later: We should get the ranking component to also allow just `showLowest`, without having to set `="true"`

**Review by juliawu — APPROVED** [review]
Thanks for the changes! This naming is much cleaner.

File: web-components/docs/README.md (hunk 1 above)
Comment (dwnoble) [reply web-components/docs/README.md]: Good catch! I went ahead and added a fix for this in this PR

**Pull request merged**